builders/pallets-precompiles/precompiles/overview.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Solidity Precompiles
+description: An overview of the available Solidity precompiles on Moonbeam. Precompiles enable you to interact with Substrate features using the Ethereum API.
+---
+
+# Overview of the Precompiled Contracts on Moonbeam
+
+![Precompiled Contracts Banner](/images/builders/pallets-precompiles/precompiles/overview/overview-banner.png)
+
+## Overview {: #introduction } 
+
+On Moonbeam, a precompiled contract is native Substrate code that has an Ethereum-style address and can be called using the Ethereum API, like any other smart contract. The precompiles provide access to Substrate pallets that are coded in Rust and not normally accessible from the Ethereum side of Moonbeam.
+
+The Substrate code responsible for implementing precompiles can be found in the [EVM pallet](/learn/features/eth-compatibility/#evm-pallet){target=_blank}. The EVM pallet includes the [standard precompiles found on Ethereum and some additional precompiles that are not specific to Ethereum](https://github.com/paritytech/frontier/tree/master/frame/evm/precompile){target=_blank}. It also provides the ability to create and execute custom precompiles through the generic [`Precompiles` trait](https://paritytech.github.io/frontier/rustdocs/pallet_evm/trait.Precompile.html){target=_blank}. There are several custom Moonbeam-specific precompiles that have been created, all of which can be found in the [Moonbeam codebase](https://github.com/PureStake/moonbeam/tree/master/precompiles){target=_blank}.
+
+The Ethereum precompiled contracts contain complex functionality that is computationally intensive, such as hashing and encryption. The custom precompiled contracts on Moonbeam provide access to Substrate-based functionality such as staking, governance, XCM-related functions, and more.
+
+The Moonbeam-specific precompiles can be interacted with through familiar and easy-to-use Solidity interfaces using the Ethereum API, which are ultimately used to interact with the underlying Substrate interface. This flow is depicted in the following diagram:
+
+![Precompiled Contracts Diagram](/images/builders/pallets-precompiles/precompiles/overview/overview-1.png)
```

Comment:
We might as well use the same person that I used in Remote EVM calls? I like that one because it has the Ledger / Metamask logo :)

Reply:
just curious, where did that person come from? haha

Reply:
I really can't remember 🤔 maybe from the polkadot decoded edits that the team did? 🤷